modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java

```diff
@@ -1182,6 +1183,14 @@ private ObjectDefinition _publishObjectDefinition(
 			throw new RequiredObjectFieldException();
 		}
 
+		count = _objectFieldPersistence.countByODI_L(
+			objectDefinition.getObjectDefinitionId(), true);
+
+		if (!objectDefinition.isEnableLocalization() && (count > 0)) {
```

Comment:
@MarinhoFeliphe I believe this validation should be made while creating localized fields.

For example, the value of the `enableLocalization` only turn into `false` when there's no localized fields (while creating/updating ObjectDefinition). If we guarantee this, we don't need this check here.

Reply:
@gabrielwas agreed!

done [here](https://github.com/liferay-objects/liferay-portal/pull/1836/commits/95bb60c3d788f1f9a903dece61af027abb195456)

thank you!